appcelerator/titanium_mobile
# [TIMOB-15021] adding ActionBar support library to enable ActionBar on all Android versions

Using the new Android Support Library to enable ActionBar across the board. 
See http://www.youtube.com/watch?v=6TGgYqfJnyc for general info on this.
The submitted pull request builds cleanly, however it currently doesn't pass testing since the required Theme.AppCompat... in the test app's AndroidManifest.xml is not found by Studio. Google's instructions for applying a support library with resources (see http://developer.android.com/tools/support-library/setup.html ) don't directly apply to Titanium SDK development, and I am not expert enough on Titanium Android build integration to fix that step. Hopefully someone with knowledge of this process can at least perform that step - this support library would be a huge improvement!

To test: (currently fails)
1. Create a new Titanium mobile project for Android
2.  Put android:theme="@style/Theme.AppCompat.Light.DarkActionBar" (for example) in AndroidManifest.xml for at least one of the app's activities.
3. Run on both API 10 and on API 11+ - should see ActionBar in all cases.
